### PR TITLE
Move flwr.datasets to baselines

### PR DESCRIPTION
**Pull request by danieljanes**

### CODE REVIEW

**Review by pedropgusmao — APPROVED** [review]
Looks good

**Comment by danieljanes** [comment]
Thanks for the help @pedropgusmao, waiting for approval from @tanertopal

**Review by tanertopal — APPROVED** [review]

**Pull request merged**
